src/main/resources/i18n/Messages_en.properties

```diff
@@ -124,6 +124,15 @@ monitoringPlotNorthwestCorner={0} Northwest Corner
 monitoringPlotSoutheastCorner={0} Southeast Corner
 # {0} is plot name
 monitoringPlotSouthwestCorner={0} Southwest Corner
+# {0} is the report prefix, {1} is Project Deal Name
+notification.acceleratorReport.submitted.app.title={0} Report Submitted for {1}
+# {0} is Project Deal Name, {1} is the report prefix
+notification.acceleratorReport.submitted.app.body={0} has submitted their {1} Report.
+# {0} is Project Deal Name, {1} is the report prefix
+notification.acceleratorReport.submitted.email.body={0} has submitted their {1} Report.
+notification.acceleratorReport.submitted.email.buttonLabel=View Submitted Report
+# {0} is the report prefix, {1} is Project Deal Name
```

Comment:
Not sure if we care much about accuracy of comments, but these will only be the Project Deal Name if it exists, else the Project Name

Reply:
I'd probably say "project name" here. These comments are written for the (mostly freelance) translators that Phrase sends our strings to. A random translator isn't going to know what a "deal name" is.